packages/dd-trace/test/opentelemetry/span.spec.js

```diff
@@ -348,12 +348,24 @@ describe('OTel Span', () => {
     }
 
     const error = new TestError()
-    span.recordException(error)
+    const datenow = Date.now()
+    span.recordException(error, datenow)
 
     const { _tags } = span._ddSpan.context()
-    expect(_tags).to.have.property(ERROR_TYPE, error.name)
+    expect(_tags).to.have.property(ERROR_TYPE, 'otel.recordException:' + error.name)
```

Comment:
This workaround is kinda funky. If the temporary workarounds aren't greawt, I think it would be OK for now if the `recordException` only creates the span event and doesn't update the error tracking tags

Reply:
Yeah it is a little funky, i updated the code to instead hack the tagger and the formatter rather then adding an identifier to the error_type property